lib/mockbuilder.py

```diff
@@ -41,25 +41,28 @@ def __init__(self, config):
         self.build_dir = None
         self.archive = None
         self.timestamp = datetime.datetime.now().isoformat()
+        self.common_mock_args = (
+            "%(binary)s -r %(config_file)s %(extra_args)s "
+            "--uniqueext %(suffix)s" % dict(
+                binary=MOCK_BIN, config_file=self.mock_config,
+                extra_args=self.mock_args, suffix=self.timestamp))
```

Comment:
I don't think we should merge all the arguments together. The binary path and config file are required and so I think each one of them deserves a separate parameter. I'm adding those to the config file. The mock_args are optional, so I'm leaving as is. The timestamp is generated dynamically, so a parameter for it does not make much sense.